### DISCUSSION TITLE
Multiple versions with subfolders

### DISCUSSION THREAD

**Discussion by tknx** [jellyfin/jellyfin#7305] [General] [answered]
I would prefer to maintain the folder names I have but still have multiple versions.

Would be great if you could have a structure like:

    Alien (1979)
    +--- Alien.RippedfromBluRay.1979.Theatrical - Theatrical/
    +--- Alien.Foundsomewherelese.1979.DC.Spanish - Director's Cut/

Right now it seems like you can only have movie files under the parent folder.

**Comment by crobibero** [comment, marked as answer]
https://features.jellyfin.org/
